src/panel-manager.js

```diff
@@ -102,8 +111,12 @@ genotet.panelManager.addPanel = function(view) {
     .appendTo('.tab-content');
   $(view).on('genotet.focus', function() {
     var clickedViewID = view.viewID_;
-    genotet.panelManager.activatePanel_(clickedViewID);
-  });
+    this.activatePanel_(clickedViewID);
+  }.bind(this));
+  this.adjustTabHeight();
+
+  // Adjust tab height.
+  $(window).resize(this.adjustTabHeight.bind(this));
```

Comment:
This doesn't seem right. You are binding window resize handler EVERY TIME a view is created. It should be binded only once.